app/__init__.py

```diff
@@ -97,4 +106,11 @@ def create_app(debug=False):
                             active_brew_sessions[uid].machine_type = mtype
                             active_brew_sessions[uid].is_pico = True if mtype in [MachineType.PICOBREW, MachineType.PICOBREW_C] else False
 
+    if sys.platform != "darwin":
+        from .main import tilt
+        if server_cfg['tilt_monitoring']:

```

Comment:
`tilt_monitoring` for those upgrading to this source will be faced with a `KeyError: 'tilt_monitoring'` here because this key doesn't exist in their local non source controlled `config.yaml` file.

```suggestion
        if 'tilt_monitoring' in server_cfg:
```

Reply:
Good catch. I hadn't though out existing users (although I did think to check this for the interval). Will fix.